tests/test_errors.py

```diff
@@ -1,46 +1,71 @@
 import os
 import platform
+from typing import TYPE_CHECKING
 import warnings
 
 from packaging.version import parse
 import pandas as pd
 from pandas import errors
-
-# TODO: Remove all imports below after switch to 1.5.x, these moved to pandas.errors
-from pandas.core.base import (
-    DataError,
-    SpecificationError,
-)
-from pandas.core.common import (
-    SettingWithCopyError,
-    SettingWithCopyWarning,
-)
-from pandas.core.computation.engines import NumExprClobberingError
-from pandas.core.computation.ops import UndefinedVariableError
-from pandas.core.indexing import IndexingError
 import pytest
 
-from pandas.io.clipboard import (
-    PyperclipException,
-    PyperclipWindowsException,
-)
-from pandas.io.formats.css import CSSWarning
-from pandas.io.pytables import (
-    AttributeConflictWarning,
-    ClosedFileError,
-    IncompatibilityWarning,
-    PossibleDataLossError,
-)
-from pandas.io.sql import DatabaseError
-from pandas.io.stata import (
-    CategoricalConversionWarning,
-    InvalidColumnName,
-    PossiblePrecisionLoss,
-    ValueLabelTypeMismatch,
-)
+PD_LT_15 = parse(pd.__version__) < parse("1.5.0")
```

Comment:
Move `PD_LT_15` to `tests/__init__.py`, then you don't have to `from packaging.version import parse` in all those files.

Reply:
Fixed.